vortex-sampling-compressor/src/constants.rs

```diff
@@ -0,0 +1,27 @@
+#![allow(dead_code)]
+pub mod depth {
```

Comment:
Because the config is `max_cost` and the fn on the `EncodingCompressor` trait is `cost`, but it's semantically measuring `depth` (and that's the term from BtrBlocks, we just twiddle it a bit because e.g., some of our encodings are structural, and we don't have another way yet to steer away from e.g., delta-of-deltas)

Happy to take suggestion on how to consolidate the naming; it represents a hard constraint to constrain the search space away from arbitrarily deep compression trees

Reply:
I’m in favour of getting rid of additional module. Either the usage site provides context or the source code comments. You wouldn’t have constants with same name in different module